Source/analyzer/path.cpp

```diff
@@ -180,43 +189,43 @@ bool IsGoodLevelSoursororStrategy()
 		int walkTicks = GetWalkTime(Spawn, StairsDown);
 		if (walkTicks == -1) {
 			if (Config.verbose)
-				std::cerr << "Path: Gave up on walking to the stairs" << std::endl;
+				std::cerr << "PATH: Aborted walking to stairs." << std::endl;
 			return false;
 		}
 		tickLenth += walkTicks;
 	} else if (currlevel == 9) {
 		LocateItem();

```

Comment:
This looks a bit broken now, it's still checking for POI and it's going to be looking for `targetStr` which the docs don't mention and it's going to assume that finding it allows the player to teleport through the rest of the game.

Reply:
How about this?